pgjdbc/src/test/java/org/postgresql/test/jdbc2/TimestampTest.java

```diff
@@ -577,30 +578,54 @@ private void timestampTestWOTZ() throws SQLException {
       assertNotNull(t);
       assertTrue(t.equals(TS1WOTZ));
 
+      tString = rs.getString(1);
+      assertNotNull(tString);
+      assertTrue(tString.equals(TS1WOTZ_PGFORMAT));
```

Comment:
>I was also tempted to refactor the surrounding code

Yes, that makes sense. However please use another PR if you perform that refactoring.